setup.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+from __future__ import print_function
+
+from glob import glob
+import os
+import sys
+
+from setuptools import setup, Extension
+from Cython.Build  import cythonize
+
+if sys.argv[-1] == 'setup.py':
+    print('To install, run \'python setup.py install\'')
+
+if sys.version_info[:2] < (2, 7):
+    print(
+        'networkx-lemon requires Python version 2.7 or later' +
+        ' ({}.{} detected).'.format(*sys.version_info[:2]))
+    sys.exit(-1)
+
+libraries = [
+    ('lemon', {'sources': glob('src/lemon/*.cc') +
+                          glob('src/lemon/bits/*.cc'),
```

Comment:
`ArgParser` is used in the `demo` and `tools` directories, which you did not include. `Color` is used in `graph_to_eps.h` and `planarity.h`, which we can ignore for now.